variables.py

```diff
@@ -62,38 +66,40 @@
 # **** Challenge 2: Problem 1 ****
 # Store the sum of num1 and num2 in a variable.
 # Print the sum.
-
```

Comment:
The first step of the problem (`# Store the sum of num1 and num2 in a variable.`) expects you to create a variable, like so:

```suggestion
num3 = num1 + num2
```